cfgov/jinja2/v1/owning-a-home/closing-disclosure/index.html

```diff
@@ -0,0 +1,101 @@
+{% extends "base.html" %}
+{% import "molecules/social-media.html" as social_media with context %}
+{% set page={"search_description":"Use this tool to double-check that all the details about your loan are correct on your Closing Disclosure. If something looks incorrect or unexpected, ask why."} %}
+{% set active_page = "closing-disclosure" %}
+{% set page_js = "closing-disclosure" %}
+{% block title %}Closing Disclosure Explainer{% endblock title %}
+{% block metadescription %}Use the CFPB"s tool to double-check that all the details about your loan are correct on your Closing Disclosure.{% endblock metadescription %}
+
+{% block banner_top %}
+{% include "_templates/brand-header.html" %}
+{% endblock %}
+
+{% block css %}
+<link rel="stylesheet" href="{{ static('oah/main.css') }}">
+{{ super() }}
+{% endblock css %}
+
+{% block javascript %}
+    <script
+  src="http://code.jquery.com/jquery-3.3.1.slim.min.js"
+  integrity="sha256-3edrmyuQ0w65f8gfBsqowzjJe2iM6n0nKciPUp8y+7E="
+  crossorigin="anonymous"></script>
+    {{ super() }}
+    <script src="{{ static('js/owning-a-home/common.js') }}"></script>
+
+{% endblock javascript %}
+
+{% block content %}
+<main class="content">
+    <div class="content_wrapper">
+        <nav class="breadcrumbs" aria-label="Breadcrumbs">
+            <a href="/owning-a-home/" class="breadcrumbs_link">Owning a Home</a>
+        </nav>
+    </div>
+    <div class="content_wrapper">
+        <div class="form-explainer content_main u-pb0">
+            <div class="page-intro content-l">
+                <div class="content-l_col-2-3">
+                    <h1 class="page-title">Closing Disclosure Explainer</h1>
+                    <p class="lead">Use this tool to double-check that all the details about your loan are correct on your Closing Disclosure. Lenders are required to provide your Closing Disclosure three business days before your scheduled closing. Use these days wisely&mdash;now is the time to resolve problems. If something looks different from what you expected, ask why.</p>
```

Comment:
Could you add some breaks to this content to make it somewhat manageable in the GH interface?